code/processes/gateway.q

```diff
@@ -409,6 +413,12 @@ getserverids:{[att]
   ];
 
   // its a dictionary of attributes
+    // check if attribute types are correct types
+    correctAttTypes:`date`servertype`tables!14 11 11h;
+    w:correctAttTypes[key att]=abs type each att;
+    if[not all w key att;
+    '"Wrong function parameter types provided.", " ",(" "sv string where not w)," parameter(s) need type(s) ",.Q.s1 correctAttTypes where not w
```

Comment:
Where's the extra indent here and on line 393? :cry:

Reply:
Sorry I thought you meant the earlier if check, will get it done now!